.travis.yml

```diff
@@ -30,8 +30,8 @@ env:
 before_install:
```

Comment:
Please add go 1.13.x to the matrix as well.

Reply:
Yes, I think we can do it explicitly too.

The `tip` version is 1.13, but it currently fails because of the vendoring issue. Ref. comment from @vrothberg I think we should just move the vendoring check to `1.13`, since that is what most users use anyways.

Reply:
Yes, in the go.mod it always references 1.13 so I think this makes sense.